packages/server/graphql/public/types/AddFeatureFlagToOrgPayload.ts

```diff
@@ -0,0 +1,15 @@
+import isValid from '../../isValid'
+import {AddFeatureFlagToOrgPayloadResolvers} from '../resolverTypes'
+
+export type AddFeatureFlagToOrgPayloadSource = {orgIds: string[]} | {error: {message: string}}
+
+const AddFeatureFlagToOrgPayload: AddFeatureFlagToOrgPayloadResolvers = {
+  organizations: async (source, _args, {dataLoader}) => {
```

Comment:
+1 see comment above. This could become `updatedOrganizations` if we return the updated `orgIds` from the mutation